docs/releases/v2.6.0/sdlc-compliance.md

```diff
@@ -0,0 +1,45 @@
+SSDLC Compliance Report: Atlas Kubernetes Operator Manager v2.6.0
+=================================================================
+
+- Release Creators: s.urbaniak@mongodb.com,igor.karpukhin@mongodb.com,helder.santana@mongodb.com,jose.vazquez@mongodb.com
```

Comment:
Unless some is co-driving the release, or shadowing or will be taking over I would suggest:
```suggestion
- Release Creators: igor.karpukhin@mongodb.com
```